SimTransport/TotemRPProtonTransportParametrization/src/TMultiDimFet.cc

```diff
@@ -152,7 +158,6 @@ const TMultiDimFet &TMultiDimFet::operator=(const TMultiDimFet &in) {
   fShowCorrelation = in.fShowCorrelation;  // print correlation matrix
   fIsUserFunction = in.fIsUserFunction;    // Flag for user defined function
   fIsVerbose = in.fIsVerbose;              //
-  return in;
```

Comment:
@cms-sw/simulation-l2 , any idea why the assignment operator was returning `in` instead of `*this` ?

Reply:
@smuzaffar , you are right this is a mistake. This may indicate that the copy constructor is never used and may be deprecated.

Reply:
@civanch , this was in assignment `operator=` . Should I fix this and return `*this`?